ui/raidboss/data/06-ew/raid/p4s.ts

```diff
@@ -111,21 +156,29 @@ const triggerSet: TriggerSet<Data> = {
     {
       id: 'P4S Kothornos Kick',
       regex: /Kothornos Kick/,
-      beforeSeconds: 5,
+      beforeSeconds: 5.3,
       infoText: (data, _matches, output) => {
-        if (data.role === 'tank' || data.role === 'healer')
-          return;
-        return output.text!();
+        let jumpDir;
+        if (data.jumpDir1 === 'east')
+          jumpDir = !data.kickTwo ? output.west!() : output.east!();
+        if (data.jumpDir1 === 'west')
+          jumpDir = !data.kickTwo ? output.east!() : output.west!();
+
+        return output.text!({ dir: jumpDir });
```

Comment:
`popup-text.ts:350 Trigger P4S Kothornos Kick has non-string param value dir.` I think this is getting called with undefined dir somehow.

Reply:
hmmm, I used the undefined of it as a false input. Perhaps that's bad design?